app/models/form526_submission.rb

```diff
@@ -258,8 +263,6 @@ def jobs_succeeded?
     a_submit_form_526_job_succeeded? && all_other_jobs_succeeded_if_any?
   end
 
-  class SubmitForm526JobStatusesError < StandardError; end
-
```

Comment:
This error isn't used anywhere.

Reply:
Heh, I would guess it has never been used anywhere, since it wasn't used anywhere when Ed [added it last year](https://github.com/department-of-veterans-affairs/vets-api/commit/eb56d40ef8ccc1fac45bda4b1c5fa0fbbbbc3b52#diff-ada3116cb66cda03a293e8c804592ccb3bc181f3e64689c66fe27d19b0d3cad1R216).